Resources/PO/TopNav.robot

```diff
@@ -0,0 +1,45 @@
+*** Settings ***
+Library  SeleniumLibrary
+
+
+*** Variables ***
+${LOGO_LOCATOR} =  //li[@class="logo"]
+${SIGNIN_LINK_LOCATOR} =  //li[@class="user-li"]/a
+${USER_MENU_LOCATOR} =  //div[@id="user-dropdown"]
+${USER_PROFILE_LINK} =  //a[@href="/home"]
+#user-dropdown-menu > li:nth-child(4) > a
+
+*** Keywords ***
+Verify Logo Loaded
+    Wait Until Page Contains Element  ${LOGO_LOCATOR}
+
+Navigate To Main Page
+    Verify Logo Loaded
+    Click Element  ${LOGO_LOCATOR}
+
+Go to Login Page
```

Comment:
different name style: 
`Navigate To Main Page` and `Go to Login Page`

when you will found this in test code first idea that those two keywords use different navigation method. 
Try to found singly style of naming and coding for similar functionality it helps a lot in supporing code in future